### ISSUE TITLE
Package & Integrate fwupd (T2532)

### ISSUE THREAD

**Issue by celticmagic** [getsolus/packages#292]
<div dir="rtl"><b>Marc&nbsp;Van&nbsp;Giel</b>&nbsp;(<em>&#35;marcvg</em>),&nbsp;2017-02-10&nbsp;14:22:26&nbsp;UTC</div><br/>

**Actionable Item:** Integrate fwupd (pulled from repo **due to lack of integration points**) in various tools and software in Solus. The should likely be:

- [x] Package fwupd and deps
- [x] ~~Software Center - Firmware installation and updating. Relevant upstream repo [here](https://github.com/solus-project/solus-sc) however no upstream issue, not necessary.~~ See #318 and #319.
- [x] Track and land upstream `clr-boot-manager` fwupd support, pending issue [here](https://github.com/clearlinux/clr-boot-manager/issues/77). PR [here](https://github.com/clearlinux/clr-boot-manager/pull/251).

**Comment by livingsilver94** [comment]
This checkbox:

> Software Center - Firmware installation and updating. Relevant upstream repo [here](https://github.com/solus-project/solus-sc) however no upstream issue, not necessary.

I'd remove it as we'll eventually deprecate SC if favor of specific DE's store frontends. Or at least rephrase it to reflect that.

**Comment by silkeh** [comment]
Closing this as fwupd has landed in https://github.com/getsolus/packages/commit/1165cadc8e411c30f3ec4a8a59795591277842d6. See #318 and #319 for software centers with integration.
